app/src/main/res/values-fr/strings.xml

```diff
@@ -55,7 +53,7 @@
     <string name="custom_input_style_note_message">"Vous devez activer votre style de saisie personnalisé avant de l\'utiliser. Voulez-vous le faire maintenant ?"</string>
     <string name="enable">"Activer"</string>
     <string name="not_now">"Pas maintenant"</string>
-    <string name="custom_input_style_already_exists">"Le style de saisie suivant existe déjà : <xliff:g id="INPUT_STYLE_NAME">%s</xliff:g>."</string>
+    <string name="custom_input_style_already_exists">Le style de saisie suivant existe déjà : <xliff:g id="INPUT_STYLE_NAME" example="English (Dvorak)">%s</xliff:g></string>
```

Comment:
Don't change the `<xliff:g id="INPUT_STYLE_NAME">%s</xliff:g>` part

Reply:
Ok will add a "No translate" label in my Weblate.